enterprise/views.py

```diff
@@ -600,6 +600,7 @@ class CourseEnrollmentView(View):
         'expected_learning_items_text': _("What you'll learn"),
         'course_full_description_text': _('About This Course'),
         'staff_text': _('Course Staff'),
+        'discount_text': _('Discount provided by')
```

Comment:
Not sure how much it matters, but we can probably format `_('Discount provided by {enterprise_customer_name}').format(enterprise_customer_name=enterprise_customer.name)` in context inside the method and make the template slightly simpler.

Reply:
It is a good idea but I added the string to `STATIC_TEXT_FORMAT` that does not have the `enterprise_customer` info in it. otherwise we need to update it out of that block

Reply:
Yep, that's what I meant -- to put it into a context where `enterprise_customer` is reachable. But the new version looks good to me :)